index.js

```diff
@@ -20,11 +27,20 @@ module.exports = function (req) {
       var opts = options || {};
       opts.expires = new Date(0);
 
-      return !!(req.res.cookie(name, '', opts));
+      try {
+          if (req) {
+              req.res.cookie(name, '', opts)
```

Comment:
Would this line ever throw, or just the `delete` below it? I like having just one statement per throw, so you don't catch unintended errors.